code/modules/mob/dead/observer/spells.dm

```
@@ -49,4 +63,7 @@ var/global/list/boo_phrases=list(
 				A:spookymode=1
 
 			if(istype(A, /obj/machinery/ai_status_display))
-				A:spookymode=1
\ No newline at end of file
+				A:spookymode=1
+
+	if (shadowling_blocked_light_flicker)
+		to_chat(action.owner.client, "The nearby shadowling laughs at your unsuccessful attempt to flicker lights in their presence.")
```

I'd just make this fail silently.